IBPSA/Airflow/Multizone/BaseClasses/TwoWayFlowElement.mo

```diff
@@ -19,13 +19,13 @@ partial model TwoWayFlowElement "Flow resistance that uses the power law"
   parameter Modelica.Units.SI.Velocity vZer=0.001
     "Minimum velocity to prevent zero flow. Recommended: 0.001";
 
-  Modelica.Units.SI.VolumeFlowRate VAB_flow(nominal=0.001)
+  Modelica.Units.SI.VolumeFlowRate VAB_flow(nominal=1/3600)
```

Comment:
None of these changes to the nominal attribute has an effect, as tested on the original model. They are not bad per se, but not necessary or helpful either. Unless there is a reason, we should not just change setting for no reason as any change indicates to others that this was needed.

Reply:
I will revert them. They are in fact not used in this example model. (The `nominal` attribute is only used for states and for iteration variables of nonlinear equations, for which we have none here.)